DEBUG.md

```diff
@@ -0,0 +1,84 @@
+# Debugging logs
```

Comment:
Do we want to call this "Debugging logs"? To me, it would make more sense to just call it `Debugging`.